scripts/qiita-test-install

```diff
@@ -238,7 +238,7 @@ class QiitaConfig(TestCase):
         if not self.psql_version:
             self.assertTrue(False, 'PostgreSQL not running or configured')
 
-        acceptable_version = (9, 1, 14)
+        acceptable_version = (9, 3, 0)
```

Comment:
This is fine, however line 245 needs to be changed as according to the ChangeLog we now have a "minimum acceptable" meaning that anything greater or equal to that should be considered valid, right?